tests/build/build_templates.go

```diff
@@ -600,14 +600,16 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 						// Refresh our copy of the PipelineRun for latest results
 						pr, err = kubeadminClient.TektonController.GetPipelineRun(pr.Name, pr.Namespace)
 						Expect(err).NotTo(HaveOccurred())
-
+						GinkgoWriter.Printf("The PipelineRun %s in namespace %s has status.conditions: \n%#v\n", pr.Name, pr.Namespace, pr.Status.Conditions)
+						
 						// The UI uses this label to display additional information.
 						Expect(pr.Labels["build.appstudio.redhat.com/pipeline"]).To(Equal("enterprise-contract"))
 
 						// The UI uses this label to display additional information.
-						tr, err := kubeadminClient.TektonController.GetTaskRunFromPipelineRun(kubeadminClient.CommonController.KubeRest(), pr, "verify")
+						tr, err := kubeadminClient.TektonController.GetTaskRunFromPipelineRun(kubeadminClient.CommonController.KubeRest(), pr, "verify")			
 						Expect(err).NotTo(HaveOccurred())
 						Expect(tr.Labels["build.appstudio.redhat.com/pipeline"]).To(Equal("enterprise-contract"))
+						GinkgoWriter.Printf("The TaskRun %s of PipelineRun %s  has status.conditions: \n%#v\n", tr.Name, pr.Name, tr.Status.Conditions)	
```

Comment:
I think we should do this right after `Expect(err)` but before any other assertions. Move this line up.